### PR TITLE
Task 6-Implement Inventory  Create MUI Modal

### PR DESCRIPTION
**Pull request by jditch**
See ticket #68 for specifications and requirements

### CODE REVIEW

File: ui/src/components/Inventory/InventoryFormModal.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+                    <Field
+                      custom={{ variant: 'outlined', fullWidth: true, }}
+                      name='bestBeforeDate'
+                      label='Best Before Date'
+                      component={TextField}
+                      type='date'
+                    />
+                  </Grid>
+                  <Grid item xs={12} sm={12}>
+                    <FormControlLabel control={<Checkbox/>} label="Never Expires" name = 'neverExpires'/>
```
Comment (lndyaouie) [inline ui/src/components/Inventory/InventoryFormModal.js]: Change this into a formik  `<Field />` using the following properties:
```
custom={{ variant: 'outlined', fullWidth: true, }}
as={FormControlLabel}
control={<Checkbox />}
name='neverExpires'
label='Never Expires'
type="checkbox"
```

File: ui/src/components/Inventory/InventoryFormModal.js [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+import DialogTitle from '@material-ui/core/DialogTitle'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import Grid from '@material-ui/core/Grid'
+import MenuItem from '@material-ui/core/MenuItem'
+import React from 'react'
+import TextField from '../Form/TextField'
+import { Field, Form, Formik } from 'formik'
+
+const validateRequired = value => (value || typeof value === 'number' ? undefined : 'Required')
+const validatePositiveNumber = value =>
```
Comment (lndyaouie) [inline ui/src/components/Inventory/InventoryFormModal.js:15]: Optional check: A null value is can be entered if the 0's are deleted.

**Review by lndyaouie — CHANGES_REQUESTED** [review]

File: ui/src/components/Inventory/InventoryFormModal.js (hunk 1 above)
Comment (jditch) [reply ui/src/components/Inventory/InventoryFormModal.js]: Change made in commit 34e7176e11f9d1cfb470e582f113750362cb1e02

File: ui/src/components/Inventory/InventoryFormModal.js (hunk 2 above)
Comment (jditch) [reply ui/src/components/Inventory/InventoryFormModal.js:15]: Implemented a check for an empty string `''` in commit 34e7176e11f9d1cfb470e582f113750362cb1e02

**Review by lndyaouie — APPROVED** [review]

**Pull request merged**